tests/pass-dep/libc/libc-eventfd.rs

```diff
@@ -227,9 +224,7 @@ fn test_two_threads_blocked_on_eventfd() {
         assert_eq!(counter, (u64::MAX - 1));
     });
 
-    thread1.thread().unpark();
-    thread2.thread().unpark();
-    thread3.thread().unpark();
+    thread::yield_now();
```

Comment:
This wouldn't even be needed. Calling `thread1.join()` will block the main thread, and thus move control to thread 1.